test/test_eos_modifiers.cpp

```diff
@@ -62,7 +62,6 @@ using variadic_utils::transform_variadic_list;
 static constexpr const auto full_eos_list = tl<IdealGas>{};
 static constexpr const auto relativistic_eos_list = tl<IdealGas>{};
 static constexpr const auto unit_system_eos_list = tl<IdealGas>{};
-static constexpr const auto apply_to_all = al<ScaledEOS, ShiftedEOS>{};
```

Comment:
I assume this is removed because we don't need it? I see we test shifted and scaled below

Reply:
Correct. Removed because of `unused variable` warning.